File: Angular/src/app/scenario.ts
import { Content } from "./content";

export class Scenario {
  public id : number;
  public name : string;
  public contentId : string;
  public image : string;
  public icon : string;
  public content : Content;

  constructor() {
    this.id = -1;
    this.name = "";
    this.contentId = "";
    this.image = "";
    this.icon = "";
    this.content = new Content();
  }
}

File: Domain/Scenario.cs
namespace Domain;

public class Scenario
{
    //public virtual ICollection<Game>? Games { get; set; }
    public int Id { get; set; }
    public string Name { get; set; }
    public int ContentId { get; set; }
    public virtual  Content? Content { get; set; }
    public string Image { get; set; }
    public string Icon { get; set; }
}